cmake/FindMFEM.cmake

```
@@ -36,11 +47,17 @@ if(MFEM_FOUND)
     add_library(XSDK::MFEM INTERFACE IMPORTED)
   endif()
 
```

For CUDA we need this:
```suggestion

  if(ENABLE_CUDA)
    string(REPLACE "-Xlinker=" "-Wl," MFEM_EXT_LIBS "${MFEM_EXT_LIBS}")
  endif()
```